lib/models/invitation.py

```diff
@@ -0,0 +1,68 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from requests import HTTPError
+
+from lib.api.checkr_api import CheckrAPI
+from lib.helpers.checkr_api_error import CheckerAPIFailedCreation
+
+
+class WorkLocation:
+    pass
+
+
+@dataclass
+class Invitation:
+    checkr_api: CheckrAPI
+    package: str
+    candidate_id: str
+    work_locations: list[dict[str, str]]
+
+    node: Optional[str] = None
+    tags: Optional[list[str]] = None
+    communication_types: Optional[list[str]] = None
+    uri: Optional[str] = None
+    object: Optional[str] = None
+    invitation_url: Optional[str] = None
+    status: Optional[str] = None
+    created_at: Optional[str] = None
+    expires_at: Optional[str] = None
+    deleted_at: Optional[str] = None
+    report_id: Optional[str] = None
+
+    id: Optional[int] = None  # From API Creation Response
```

Comment:
I'm not sure if I need to use Optional in every field that is optional or if it's optional.

Reply:
If it is None or something else, you always need the Optional. If it is just `None` you don't need the optional, just `None` will do.

Reply:
I get it now. Thank you!